.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,35 +1,9 @@
+### Change Description
```

Comment:
Can we please remove this header? Upon creating a PR, GitHub will place the commit message's body at the top. So the first manual step is to move this header.

Reply:
I think the template should either not have any headers or use multiple to bring structure to the PR/the commit message. I don't think there would be anything wrong with having sections in the commit message if that makes it more readable.